### ISSUE TITLE
Popup gets cropped on smaller resolution displays and depending on toolbar button placement

### ISSUE THREAD

**Issue by heforfree** [futpib/policeman#40]
as you can see some optons not visible when all content type enabled
if you can make some how position of this button Dynamic
or you can change font size to 11px
with Stylish i set below style and everything show fine
#policeman-popup {
font-size:11px;
}

edit:change font size work but if i change location of policeman toolbar icon it not working
edit2:i set 10.9px And show fine again.i put policeman toolbar icon to addressbar with   url-addon-bar
so it need smaller font

![image 1](https://cloud.githubusercontent.com/assets/2247864/4875076/d6396f7a-6281-11e4-9044-e05c6de29d77.png)

**Comment by heforfree** [comment]
@futpib 
for0.18pre5 
comparing below screenshot with first post screenshot it is really better.at least on my screen resolution
![image 1](https://cloud.githubusercontent.com/assets/2247864/5694083/c5f8b746-9955-11e4-8ab0-cd07f992e6a6.png)

and after i click on other request it is look below and it is really smaller.and good for smaller resolution
![image 2](https://cloud.githubusercontent.com/assets/2247864/5694099/f06a4b6a-9956-11e4-8710-7640b0e6d1c7.png)

Uncategorized is show as ??? if you want rename it to N/A
and limit Length of rejected request,allowed request,so popup dont fill screen


i also see this in a log
TypeError: elements[low].getBoundingClientRect is not a function scrollbox.xml:387

**Comment by futpib** [comment]
Yes, n/a is definitely better. That log line looks unrelated to policeman.

**Comment by heforfree** [comment]
see yellow high light part of screenshot it is clickable.
![image 3](https://cloud.githubusercontent.com/assets/2247864/5694160/b9584844-995a-11e4-95be-2ec11b8d650e.png)

**Comment by futpib** [comment]
So I think the last part of this issue is the long domain names problem (like with 3.1415... domain above). Tell me if I'm wrong.

**Comment by Halibut80** [comment]
Maybe, this better for buttons:
<code>display:inline</code> for <code>#policeman-popup-rejected-requests-filters-container</code> and for <code>#policeman-popup-allowed-requests-filters-container</code>
<img src="http://lh6.ggpht.com/-J24v9c2FIMY/VLH2h7GpQOI/AAAAAAAAacw/z_KuRGDxyQs/s0/clipboard.png">
without "short" name.

**Comment by heforfree** [comment]
@futpib 
i use below style to limit long url in rejected request allowed request.so fix this and then the only problem is long domain names.
https://gist.github.com/heforfree/1b0ef5477c8e01707631

![image 1](https://cloud.githubusercontent.com/assets/2247864/5694995/6af104b0-999c-11e4-9723-d3dbdcf4b6ae.png)

**Comment by futpib** [comment]
@heforfree I don't get what you think is a problem, long links are elided without any external style.

**Comment by futpib** [comment]
@Halibut80 Actually, I think this should be the default

**Comment by heforfree** [comment]
@futpib
yes it is not problem but all i say  it is better the popup dont fill whole screen.

**Comment by xarragon** [comment]
I also experienced these issues on my older Thinkpad with a XGA (1024x768) resolution and complex URLs. Version 0.18.1.1. I would just add horizontal scrollbars if the size is too small. Otherwise I love this extension, far surpasses RequestPolicy in usefulness because of the excellent panel. I can leave a more detailed bug report with a screencapture showing the issues if needed.
